backend/public-graph/graph/resolver.go

```diff
@@ -2971,12 +2978,11 @@ func (r *Resolver) ProcessPayload(ctx context.Context, sessionSecureID string, e
 		(sessionObj.Processed != nil && *sessionObj.Processed) ||
 		(sessionObj.ObjectStorageEnabled != nil && *sessionObj.ObjectStorageEnabled) ||
 		(sessionObj.Chunked != nil && *sessionObj.Chunked) ||
-		(sessionObj.Excluded != nil && *sessionObj.Excluded) ||
 		(sessionHasErrors && (sessionObj.HasErrors == nil || !*sessionObj.HasErrors))
 
-	if doUpdate {
+	if doUpdate && !excluded {
```

Comment:
Because we don't update for excluded sessions, we'll cut down on unnecessary writes on the session object.

Per this query:

```sql
select id, excluded_users
from projects
where excluded_users is not null
and excluded_users != '{}'
```

we have 42 projects using this feature.

Reply:
Not sure how much of a difference it'll make, but we can look at the p99 (or max) latency for this [trace](https://app.datadoghq.com/apm/traces?query=service%3Aworker-service%20operation_name%3AprocessPublicWorkerMessage&cols=core_service%2Ccore_resource_name%2Clog_duration%2Clog_http.method%2Clog_http.status_code&historicalData=true&messageDisplay=inline&sort=desc&spanType=all&start=1682026001766&end=1682630801766&paused=false) after this is rolled out.